pkg/tracingpolicy/tracingpolicy.go

```diff
@@ -18,6 +18,8 @@ type TracingPolicy interface {
 	TpSpec() *v1alpha1.TracingPolicySpec
 	// TpInfo returns a description of the policy
 	TpInfo() string
+	// TpAuthor  returns the author of the policy
+	TpAuthor() string
```

Comment:
Just a first (quick) comment :)

What's the motivation for adding an author at the interface? 
Generally speaking I'd prefer the to keep the interface simple.

Reply:
> Just a first (quick) comment :)
> 
> What's the motivation for adding an author at the interface? Generally speaking I'd prefer the to keep the interface simple.

I agree, the aim is to print the author of the policy: give credit to users or identify policies, but maybe we can do it without adding a new function or return author part of info?

Reply:
We can identify policies by their name. I'd leave the author field out for now unless a user requests it.

Reply:
> We can identify policies by their name. I'd leave the author field out for now unless a user requests it.

Sure but author also helps to identify it (who did write it) this PR ads possibility to load from multiple directories..., so what we can do is add an annotation method which is already part of the k8s spec anyway without detailing the fields.

The annotations[author] is optional as it is now, and users can use it if they want.

About leaving that as a request for users, the other way around is better, be proactive , put some examples as we are trying todo, document it, etc which should allow users and projects to take advantage of it...

Reply:
made it generic k8s annotations logic , then inside our helpers and log messages we can print the author field if set.

Reply:
I still don't see a use-case for this. We cannot copy all the information from the tracing policy anyway. I'd rather not add more methods to the tracing policy interface.